sw/otbn/crypto/p384_base_mult.s

```diff
@@ -19,12 +19,12 @@
  * Sets up context and calls the internal scalar multiplication routine.
  * This routine runs in constant time.
  *
- * @param[in]  dmem[0]: dptr_d0, pointer to location in dmem containing
- *                      1st private key share d0
- * @param[in]  dmem[4]: dptr_d1, pointer to location in dmem containing
- *                      2nd private key share d1
- * @param[in]  dmem[20]: dptr_x, pointer to result buffer for x-coordinate
- * @param[in]  dmem[24]: dptr_y, pointer to result buffer for y-coordinate
+ * @param[in]       x17: dptr_d0, pointer to location in dmem containing
+ *                                1st private key share d0
+ * @param[in]       x19: dptr_d1, pointer to location in dmem containing
+ *                                2nd private key share d1
```

Comment:
nit: I'd maybe just change the code below to load the addresses from `dmem[d0]` and `dmem[d1]` (that is, put the `la` instructions here instead of in the wrapper code).

Reply:
You're right, makes it more clear. Thx!!